components/feature/top-sites/src/androidTest/java/mozilla/components/feature/top/sites/OnDevicePinnedSitesStorageTest.kt

```diff
@@ -27,7 +27,7 @@ import java.util.concurrent.Executors
 private const val MIGRATION_TEST_DB = "migration-test"
 
 @Suppress("LargeClass")
-class PinnedSitesStorageTest {
+class OnDevicePinnedSitesStorageTest {
```

Comment:
This undoes a previous change. Turns out we have a redeclaration warning for PinnedSitesStorageTest because we have another PinnedSitesStorageTest.